common.mk

```diff
@@ -6,7 +6,7 @@ SHELL=/bin/bash
 ifndef RISCV
 $(error RISCV is unset. You must set RISCV yourself, or through the Chipyard auto-generated env file)
 else
-$(info Running with RISCV=$(RISCV))
+$(info #Running with RISCV=$(RISCV))
```

Comment:
Is there a reason to remove this? We used to run into lots of issues when RISCV is set to an unintended path.

Reply:
I'm not removing it.  I'm only prepending the message with the shell comment character so that when I do my thing to convert makefile into a runscript to send to a remote host, the info string doesn't get interpreted as a command. See https://github.com/firesim/firesim/pull/651/files#diff-d3631732b22e280769ab899d454c5e77021c7835748f416f2f4b76a7cfa3eb0aR235-R250

Reply:
Oh I see. Misread this line.